.circleci/config.yml

```diff
@@ -264,7 +264,7 @@ jobs:
           at: .
       - run:
           name: Test
-          command: yarn test --coverage --runInBand --core
+          command: yarn test --coverage --w2 --core
```

Comment:
This speeds up the processing of coverage a lot. It's limited to 2 CPUs since that seems to work on CIs best.